src/ios/CDVWKInAppBrowser.m

```diff
@@ -132,7 +129,11 @@ - (void)openInInAppBrowser:(NSURL*)url withOptions:(NSString*)options
         NSDate* dateFrom = [NSDate dateWithTimeIntervalSince1970:0];
         [dataStore removeDataOfTypes:[WKWebsiteDataStore allWebsiteDataTypes] modifiedSince:dateFrom completionHandler:^{
             NSLog(@"Removed all WKWebView data");
-            self.inAppBrowserViewController.webView.configuration.processPool = [[WKProcessPool alloc] init]; // create new process pool to flush all data
+
+            // Create a new process pool to flush all data.
+            WKProcessPool *newProcessPool = [[WKProcessPool alloc] init];
+            self->_processPool = newProcessPool;
+            self.inAppBrowserViewController.webView.configuration.processPool = newProcessPool;
```

Comment:
Not sure if this code is even relevant to us as I couldn't get it to execute while playing around with the app. But I wanted to maintain the existing behaviour in any case.